packages/cli/src/lib/codegen.ts

```diff
@@ -246,67 +249,119 @@ async function generateDefaultConfig(
     legacy: false,
   }).catch(() => undefined);
 
+  // Known project for SFAPI
   const sfapiSchema = getSchema('storefront');
   const sfapiProject = findGqlProject(sfapiSchema, gqlConfig);
 
-  const defaultGlob = '*!(*.d).{ts,tsx,js,jsx}'; // No d.ts files
-  const appDirRelative = relativePath(rootDirectory, appDirectory);
-
+  // Known project for SFAPI
```

Comment:
Not sure if I'd call this SFAPI. Maybe we could drop these comments since we've got the `isKnownSchema()` function?

Reply:
Ah forgot to update the comment after copy-pasting, this should be CAAPI but yeah we can drop them.